pallets/bridge/src/lib.rs

```diff
@@ -385,92 +264,12 @@ pub mod pallet {
 		pub fn remark(
```

Comment:
Is this method really useful?

Reply:
This function allows to trigger a bridge inbound call that is successful. Is not very used though, but not sure if we need to worry about it.